docs/semgrep-ci/sample-ci-configs.md

```diff
@@ -10,6 +10,29 @@ import MoreHelp from "/src/components/MoreHelp"
 
 The sample configuration files below run Semgrep CI on various continuous integration providers.
 
+## Feature support
+
+Support for certain features of Semgrep App depend on your CI provider or source code management tool (SCM). The following table breaks down the features and their availability:
+
+| Feature | GitHub | GitLab | BitBucket | CI Provider support |
+| ------- | -------- | ------- | -------- | ---------------- |
+| **Diff-aware scanning** | ✅ Yes | ✅ Yes | ✅ Yes  | ✅ Available (may need additional set up) | 
+| **Hyperlinks** | ✅ Yes | ✅ Yes | ✅ Yes  |  ✅ Available (may need additional set up) |
+| **SCM security dashboard** |  ✅ GitHub Advanced Security Dashboard |  ✅ GitLab SAST Dashboard | ❌ No | ❗ Only GitHub Actions and GitLab CI/CD |
+| **PR or MR comments in Semgrep App** |  ✅ Yes | ✅ Yes | ❌ No | ✅ CI provider agnostic; feature support is dependent on SCM |
+*Table 1.* List of features and supported SCMs and CI providers.
```

Comment:
Is a line break needed above this so that the “Table 1.” line isn’t grouped into the table above? That is what I saw in the GH preview.

Reply:
It builds correctly in docusaurus with or without the linebreak, but I added the linebreak in anyway.

Reply:
Annnnnd the linebreak does nothing to fix the github preview 😅 I think it should be ok as it builds correctly in docusaurus.